packages/style-engine/class-wp-style-engine.php

```diff
@@ -136,6 +136,20 @@ final class WP_Style_Engine {
 				),
 			),
 		),
+		'dimensions' => array(
+			'minHeight' => array(
+				'property_keys' => array(
+					'default' => 'min-height',
+				),
+				'path'          => array( 'dimensions', 'minHeight' ),
+				'classnames'    => array(
+					'has-$slug-font-size' => 'font-size',
+				),
+				'css_vars'      => array(
+					'spacing' => '--wp--preset--spacing--$slug',
+				),
```

Comment:
> The font size class name looks like a copy-and-paste error.

Good spotting. I don't think we need classnames here. Removed. 

> I take it min-height support is leveraging spacing presets, and that isn't another leftover?

Oh yeah, will it? I think it's safe to leave as it'll only output the CSS var if we ask it to by sending a `var:preset|spacing|$slug` string. Otherwise it'll work with the raw, custom values.